fuo_xiami/api.py

```diff
@@ -10,7 +10,7 @@
 logger = logging.getLogger(__name__)
 
 site_url = 'http://www.xiami.com'
-api_base_url = 'http://h5api.m.xiami.com'
+api_base_url = 'https://acs.m.xiami.com'
```

Comment:
I've found the following APIs don't work well with the new URL: 
* playlist_detail_v2
* user_favorite_artists

Reply:
Perhaps we need two URLs then? Searching is completely broken here with the old url.